Expecto/Expecto.fs

```diff
@@ -291,19 +303,22 @@ module Impl =
     >> setField "errored" errored
     >> setField "erroredCount" (align erroredCount 0)
 
-  let createSummaryText (summary: TestRunSummary) =
-    createSummaryMessage summary Info
+  let createSummaryText splitter (summary: TestRunSummary) =
```

Comment:
It's a bit unclear what a splitter is?

Reply:
I changed the name to `joinBy` since the parameter means a sign by which we would join names of a test/test case/test list etc.